scenes/menus/storybook/storybook.tscn

```diff
@@ -67,6 +74,14 @@ size_flags_vertical = 1
 text = "by Jane Bloggs, Erika Mustermann and Jóna Jónsdóttir"
 autowrap_mode = 2
 
+[node name="Animation" type="TextureRect" parent="PanelContainer/HBoxContainer/RightPage/VBoxContainer"]
```

Comment:
Because we generally have a 200% scale factor in the game (c.f. #638) this ends up looking a little small. It's not completely trivial to scale a control but I think we would do it by setting a custom minimum size to 2× the width & height of the texture.

Reply:
Yes, for TextureRect that's how you adjust the size too. By the way this is a good video explanation https://www.youtube.com/watch?v=5Hog6a0EYa0&t=529s . I come back to that video every time I use Control nodes, I wish it was a written tutorial :)

Reply:
Great reference, thanks!